patches/server/0738-Fix-entities-glitching-through-blocks-pushed-by-pist.patch

```diff
@@ -0,0 +1,76 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Peter Svensson <petersv@psv.nu>
+Date: Mon, 2 Aug 2021 12:06:05 +0200
+Subject: [PATCH] Fix entities glitching through blocks pushed by pistons
+ (#5189)
+
+Fix #5189 where entities in slime blocks are unreliable.
+
+Add a general mechanism where the entity knows how many ticks into
+the future that it has to be fully ticked. ActivationRange respects
+that and does not cull any of those ticks to the entity.
+
+diff --git a/src/main/java/net/minecraft/world/entity/Entity.java b/src/main/java/net/minecraft/world/entity/Entity.java
+index 896d892237b29eb404398db07264eb6f04786754..2f10a937accd4f791ef8ffe924a77e8ab6f86e9b 100644
+--- a/src/main/java/net/minecraft/world/entity/Entity.java
++++ b/src/main/java/net/minecraft/world/entity/Entity.java
+@@ -690,6 +690,11 @@ public abstract class Entity implements Nameable, EntityAccess, CommandSource, n
+ 
+         this.firstTick = false;
+         this.level.getProfiler().pop();
++        // Paper start   - track when all ticks are required for the entity, e.g. after a piston push
```

Comment:
What's with the extra spaces all over the place?

Reply:
Which extra spaces are you referring to? The ones inside the //Paper comments or are there others I should fix? 
Should I fix them on my side?

Reply:
Yeah, the ones inside the paper comments. If you feel like fixing them, sure, otherwise we'll take care of it on merge (though it'd be kindest if you did 😄).

Reply:
Updated in the new commit

Reply:
You have several instances of this :)

Reply:
I fixed the two instances of extra spaces I found. Were there any more?